tests-hivemq-platform-operator/src/integrationTest/java/com/hivemq/helmcharts/single/HelmInitContainerIT.java

```diff
@@ -55,16 +51,16 @@ void withAdditionalInitContainer_hivemqRunningWithVolumeMounts() throws Exceptio
     }
 
     private void assertThatFileContains(final @NotNull Pod pod) {
-        final String fileStr = "/init-container-test/init-container-test.txt";
+        final String initContainerTextFile = "/init-container-test/init-container-test.txt";
         final String expectedContent = "test init container";
 
         try (InputStream in = client.pods()
                 .inNamespace(namespace)
                 .withName(pod.getMetadata().getName())
-                .file(fileStr)
+                .file(initContainerTextFile)
                 .read()) {
             final String foundContent = new String(in.readAllBytes(), StandardCharsets.UTF_8);
-            assertThat(foundContent).contains(expectedContent);
+            assertThat(foundContent).isEqualTo(expectedContent);
```

Comment:
Does this work? In my tests the actual file had an additional line break.

Reply:
it worked, I changed from 

```
echo "test init container" >> /init-container-volume-test/init-container-test.txt
```

to 

```
echo -n "test init container" >> /init-container-volume-test/init-container-test.txt
```

So no newline is added to the txt file.